packages/lib/src/gfx/shaders/UberMaterial.js

```diff
@@ -73,84 +73,8 @@ const uberOptionNames = [
   'srcTexelSize',
 ];
 
-function UberMaterial(params) {
```

Comment:
This diff is difficult to follow. I hope you included the fix I created for a similar change in the `master` branch. There was a not quite correct migration to classes.

Reply:
When I made the change I followed your fix in the `master` branch.